### ISSUE TITLE
Not Work

### ISSUE THREAD

**Issue by UtkuGARIP** [austencm/youtube-auto-like#58]
__________________________
### Log
Fri Jan 15 2021
Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36
YouTube variant: material
navigated: https://www.youtube.com/watch?v=L6TD347Bx1k
loading options...
...options loaded (disabled: false,like_what: subscribed,like_when: instantly)
liker initialized
waiting for buttons...
...buttons ready
user not subscribed
stopped
liker initialized
waiting for buttons...
...buttons ready
user not subscribed
stopped
liker initialized
not a watch page
stopped

**Comment by markygnlg** [comment]
4 things you might want to test/try:
'
- there is now a v2.7.1 which was released about 24hrs ago as of this writing:
https://github.com/austencm/youtube-auto-like/releases
'
- secondly, I too found it 'not working' (with the 50% setting) but if I pressed 'F5'/refresh or opened the video in another tab (same as refreshing the page) the like button would be lit up: ie. it works but the graphics of the thumb only displays if you press refresh (I'm using v2.7.1 on Opera)
'
- try testing with 'Instantly'
'
- updates default to "Only videos from subbed channels" - it's working for me on all videos after I changed the setting to "ALL videos"
'
P.S.
 Sadly, Chrome removed this extension for violating YouTube's terms of service (specifically the part about causing inaccurate measurements of user engagement). [from https://github.com/austencm/youtube-auto-like ... or click on the like "CODE" at the top of the page]
